packages/amplication-client/src/util/analytics.ts

```diff
@@ -1,8 +1,9 @@
 import * as reactTracking from "react-tracking";
 import { REACT_APP_AMPLITUDE_API_KEY } from "../env";
+import { AnalyticsEventNames } from "./analytics-events.types";
 
 export interface Event {
-  eventName: string;
+  eventName: AnalyticsEventNames | string;
```

Comment:
Can we remove the alternative `string` type now that we have agreed and addressed all the scenario?

Reply:
right!